packages/cli/src/commands/account/new.ts

```diff
@@ -1,52 +1,61 @@
-import { ec as EC } from 'elliptic'
-import Web3 from 'web3'
+import { generateKeys, generateMnemonic, MnemonicLanguages } from '@celo/utils/lib/account'
+import { privateKeyToAddress } from '@celo/utils/lib/address'
+import { flags } from '@oclif/command'
 import { LocalCommand } from '../../base'
 import { printValueMap } from '../../utils/cli'
-import { ReactNativeBip39MnemonicGenerator } from '../../utils/key_generator'
 
 export default class NewAccount extends LocalCommand {
   static description =
-    'Creates a new account locally and print out the key information. Save this information for local transaction signing or import into a Celo node.'
+    "Creates a new account locally using the Celo Derivation Path (m/44'/52752'/0/0/indexAddress) and print out the key information. Save this information for local transaction signing or import into a Celo node. Ledger: this command has been tested swapping mnemonics with the Ledger successfully (only supports english)"
 
   static flags = {
     ...LocalCommand.flags,
+    password: flags.string({
+      description: 'Choose a password to generate the keys',
```

Comment:
Should this be optional?

Reply:
It's optional. `@oclif`  has a `required` field that is false for default

Reply:
Aw, I understand. The old version didn't have the option. This is more of how the user want to use it